Mapping_in_R.rmd

```diff
@@ -0,0 +1,149 @@
+# Mapping in R
+
+Hanjun Li and Chengchao Jin
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+### Overview
```

Comment:
Please change all the ### headers to ## so you're not skipping from # to ## -- it will look better that way when rendered.

Reply:
Hi Professor, we just changed all ### headers to ##. Please check again, thank you.

Reply:
Hi, I am also wondering how should I remove .ds_store as I did not remember adding this file.